kairon/async_callback/scheduler.py

```
@@ -0,0 +1,159 @@
+    if not data:
+        data = {}
+
+    data['bot'] = bot
+    data['event'] = _id
+
+    callback_config = callback.find_one({"bot": bot, 'name': schedule_action})
+
+    script = callback_config['pyscript_code']
+
+    executor_type = os.getenv('EVENTS_EXECUTOR_TYPE', 'aws_lambda')
+    if executor_type not in __executors.keys():
```

Ruff (0.8.2)

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)